cli/dev/DevServer.ts

```diff
@@ -8,6 +8,8 @@ import path from "node:path"
 import fs from "node:fs"
 import type { FileUpdatedEvent } from "lib/file-server/FileServerEvent"
 import * as chokidar from "chokidar"
+import { isFileImportsTyped } from "lib/dependency-analysis/isFileImportsTyped"
```

Comment:
This name is a bit confusing to me, what do you think of this?
```suggestion
import { isFileTypesImported } from "lib/dependency-analysis/isFileTypesImported"
```

Reply:
update, can now it be merged?

thanks